enterprise/api/renderers.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+"""
+Renderer(s) for api response.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from rest_framework.negotiation import DefaultContentNegotiation
+from rest_framework.settings import api_settings
+
+
+class IgnoreClientContentNegotiation(DefaultContentNegotiation):
```

Comment:
What does this class do? What does "IgnoreClient" mean? I think the docstrings could have been better here.

Reply:
It was referenced in [django rest framework](http://www.django-rest-framework.org/api-guide/content-negotiation/#custom-content-negotiation).

Reply:
In that case they were skipping all client requested negotiation and just returning the first. This one introduced allows the format_suffix. Maybe this should be renamed to something like `AllowSuffixContentNegotiation` or more generic `CustomContentNegotiation` 

Good catch @douglashall

Reply:
I will update the naming issue soon. thanks

Reply:
Why do we need to override here at all? Shouldn't the `DefaultContentNegotiation` class do what we need to do? What exactly is this override making possible that the `DefaultContentNegotiation` class does not? If we do need to override, shouldn't we also be overriding `select_parser`?

Reply:
From reading the code, my understanding was that this would either handle the format_suffix passed in or pull it from the query params. Since most APIs aren't use to accessing `v1/organizations.xml/` but `?format=xml` is more common this seems like a good override (if my assumptions are correct)

Its still not clear to me if an `accepts` header is handled natively in DRF or if we would need to also sent render_format based on header. I would assume (hope) that DRF handles that negotiation out-of-box. @asadiqbal08  did you test that?

Reply:
@georgebabey you are right with the above statement. Further I am sharing my findings that were examined during implementation. 

- DRF was ignoring the order in [render_classes](https://github.com/edx/edx-enterprise/pull/282/files#diff-4495418c366e5831b5e2daaea415aa56R305), may be due to djangorestframework-xml library. It is giving precedence to xml always. so that’s why I override the method. 

- `?format=json` was not working, DRF returns the xml response ignore the url format pattern but working fine when add the `.xml` and `.json` suffix in URL.

- I did not override the parser, actually, REST framework examine the Content-Type header on the incoming request, and determine which parser to use to parse the request content. Please verify the behavior on [business-sandbox](https://business.sandbox.edx.org/enterprise/api/v1/enterprise_catalogs/)  and let me know If there is something that I missed in PR. 

Many Thanks.

Reply:
We should make note of these things in the docstring. When I read the DRF code it seemed like all of this should already by handled without customization.